polyglot/piranha/test-resources/java/new_line_character_used_in_string_literal/expected/SomeClass.java

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2022 Uber Technologies, Inc.
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+ * except in compliance with the License. You may obtain a copy of the License at
+ *
+ * <p>http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * <p>Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.uber.piranha;
+
+class SomeClass {
+  
+  void someMethod(String s){      
+      assert("Hello \n World".equals(s));
```

Comment:
Presumably, without the change, this becomes:

```
  void someMethod(String s){      
      assert("Hello \\n World".equals(s));
```

correct?

Or is it:

```
  void someMethod(String s){      
      assert("Hello 
 World".equals(s));
```

?

Reply:
yes. It's the later.